### ISSUE TITLE
Cara mengganti metode bayar jika sudah terlanjut memilih?

### ISSUE THREAD

**Issue by rizalahmaddd** [Midtrans/midtrans-php#92]
halo semuanya.
saya menggunakan snap untuk pembayaran user berjalan dengan baik, namun ada kejadian user awal memilih pembayaran indomaret misalnya, lalu dia ingin mengganti ke yang lain. bagaimana ya caranya? di snap tidak ada menu untuk mengganti cara bayarnya jika sudah terlanjur dipilih, terimakasih

**Comment by rmdhfz** [comment]
Setahu saya perlu dibatalkan dulu transaksinya mas @rizalahmaddd , baru bisa dipilih metode lainnya

**Comment by uziwuzzy** [comment]
betul, flow pembayaran via midtrans itu satu arah, jadi setelah memilih hanya bisa dibatalkan atau lanjut pembayaran, jadi sekiranya salah memilih harus cancel transaksinya dulu.
